Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -338,5 +411,104 @@ protected override void OnGotFocus(RoutedEventArgs e)
             _parentMenu.SelectedMenuItem = this;
             base.OnGotFocus(e);
         }
+
+        internal void Underline()
+        {
+            if (_originalHeader == null)
+            {
+                return;
+            }
+
+            var underlineCharacterIndex = _originalHeader.IndexOf(UnderlineCharacter);
+
+            var underlinedCharacter = _originalHeader[underlineCharacterIndex + 1];
+            var text = new TextBlock();
+
+            var runWithUnderlinedCharacter = new Run
+            {
+                Text = underlinedCharacter.ToString(),
+                TextDecorations = Windows.UI.Text.TextDecorations.Underline
```

Comment:
[TextDecorations](https://docs.microsoft.com/en-us/uwp/api/windows.ui.text.textdecorations) is only available in CU, it will crash in AU or older. Maybe do a check before using?